html5/js/Client.js

```diff
@@ -330,7 +330,7 @@ class XpraClient {
     const screen_element = jQuery("#screen");
     screen_element.mousedown((e) => this.on_mousedown(e));
     screen_element.mouseup((e) => this.on_mouseup(e));
-    document.getElementById("screen").addEventListener("mousemove", (e) => this.on_mousemove(e));
+    screen_element.mousemove((e) => this.on_mousemove(e))
```

Comment:
This was changed recently in 7287ce7bd18db3ac202aa9c161142eb894c4529f to support pointer lock: #335
It's probably best to ask @CorbinWunderlich why that is, to avoid breaking things.

Reply:
Alright, it just didn't make sense to my why it was different when I came across it - not much point in the change other than that

Reply:
@whoactuallycares Can you undo this please?
@CorbinWunderlich does that make sense?

Reply:
Undone [here](https://github.com/Xpra-org/xpra-html5/pull/363/commits/c65b63b14e93b0c7688669bd9b2cf5873bc7262f)